### ISSUE TITLE
eval mode for pytorch implementation

### ISSUE THREAD

**Issue by stebuechho** [Cerebras/online-normalization#22]
Hi there,

thanks for this great package! This seems to make much more sense than batch norm and I would really like to try using OnlineNorm in some of my projects. 
But does this work properly in eval mode? What i mean is, when i put my network into eval mode with model.eval(), do you make sure the running exponential averages are not updated, to make sure no data from the test set can bleed into the model parameters?

From looking at the code in Norm2dBatched (which is now deprecated in favor of Norm2d, i guess?), it seems that the running averages are updated no matter what, since, self.mu and self.var are overwritten in the forward pass. I couldn't really figure it out in Norm2d though.

On a side note: in the docstrings of OnlineNorm1d and OnlineNorm2d, it says ecm='ls' would be default, when it's actually ecm='ac' in the default kwargs. What is the better option in your opinion?

**Comment by stebuechho** [comment]
I simply checked (should just have done that before opening an issue, sorry) and as OnlineNorm2d.norm.m and OnlineNorm2d.norm.var do not change when in eval() mode, while they do change in train mode when calling forward, this works as it should! 
Closing
